python/interpret_community/explanation/explanation.py

```diff
@@ -27,12 +27,12 @@ class BaseExplanation(ChainedIdentity):
 
     """The common explanation returned by explainers.
 
-    :param method: The explanation method used to explain the model (e.g. SHAP, LIME).
+    :param method: The explanation method used to explain the model (e.g., SHAP, LIME).
     :type method: str
-    :param model_task: The task of the original model i.e. classification or regression.
+    :param model_task: The task of the original model i.e., classification or regression.
```

Comment:
this comma after i.e. and e.g. is really bothering me hehe :), I never do that.

Reply:
If it's really bothering you, I can go either way on it. :-)  I'm trying to make your life easier.